### PR TITLE
Feature 54339 fix api user bug

### PR DESCRIPTION
**Pull request by LBruegger**

### CODE REVIEW

File: app/assets/javascripts/modules/api_user_toggle.js.coffee [hunk 1]
```diff
@@ -24,13 +24,13 @@ class app.ApiUserToggle
   lockApiUser = (id) ->
     $.ajax({
       method: 'POST',
-      url: '../api/api_users/' + id + '/lock'
-    })
+      url: '../api/api_users/' + id + '/lock',
```
Comment (TheWalkingLeek) [inline app/assets/javascripts/modules/api_user_toggle.js.coffee:27]: Won't need this `,` right? :)

File: app/assets/javascripts/modules/api_user_toggle.js.coffee [hunk 2]
```diff
@@ -24,13 +24,13 @@ class app.ApiUserToggle
...
+    });
 
   unlockApiUser = (id) ->
     $.ajax({
       method: 'DELETE',
-      url: '../api/api_users/' + id + '/lock'
+      url: '../api/api_users/' + id + '/lock',
```
Comment (TheWalkingLeek) [inline app/assets/javascripts/modules/api_user_toggle.js.coffee:33]: same here

File: app/assets/javascripts/modules/api_users.js.coffee [hunk 3]
```diff
@@ -23,10 +23,7 @@ class app.ApiUsers
     appendRow(row)
 
   apiUsersData = (data) ->
-    apiUsers = data['data']
-    if(!apiUsers)
-      apiUsers = [data['data']]
-    apiUsers
+    data['data']
```
Comment (TheWalkingLeek) [inline app/assets/javascripts/modules/api_users.js.coffee:26]: You tested this thoroughly right? This could break a couple of things i imagine

Comment (LBruegger) [reply app/assets/javascripts/modules/api_users.js.coffee:26]: Yes, I think it is thoroughly tested. I assume that every test case is covered in this [test](https://github.com/puzzle/cryptopus/blob/feature-54339-fix-api-user-bug/spec/controllers/api/api_users/lock_controller_spec.rb) as well in this [one](https://github.com/puzzle/cryptopus/blob/feature-54339-fix-api-user-bug/spec/controllers/api/api_users_controller_spec.rb). The Problem here was that the JSON structure didn't match the parameter of the [apiUsersContent](https://github.com/puzzle/cryptopus/blob/a0a66d32910a6249a610945e78656a026185c121/app/assets/javascripts/modules/api_users.js.coffee#L17) Method. I fixed this by adding Brackets to the loaded data, when a new ApiUser gets created.

**Pull request merged**